tests/datavolume_test.go

```diff
@@ -1082,39 +1083,57 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			Expect(err).ToNot(HaveOccurred())
 			f.ForceBindIfWaitForFirstConsumer(pvc)
 
-			dataVolume := utils.NewCloningDataVolume(dataVolumeName, "1Gi", pvc)
-			Expect(dataVolume).ToNot(BeNil())
+			By("Trying to create cloned DV a couple of times until we catch the elusive pods")
```

Comment:
Looks ok, but why instead of this rat race after our pods not simply pass them time-to-live-after-completion (default 0) to make such tests a bit cleaner?

Reply:
I think that having it for non-testing purposes will slow us down - any time we spent needlessly holding onto PVC means virt can't pick up the PVC.
As a testing only change, maybe

Reply:
Of course only for testing, as we spend a lot of test code lines on flakiness workarounds such as this PR, which has nothing to do with a real flow.

Reply:
I do not like it, but is not totally bad, 
Is this test failing often?

Reply:
Looking at flakefinder, it hasn't shown up in a while, but neither has anything else. I suspect something is up with flakefinder (maybe it's confused by the master->main rename)

Reply:
Let me ask the flakefinder author.

Reply:
we possibly need this change: https://github.com/kubevirt/project-infra/pull/1075

Reply:
it's flaking [fairly often](https://storage.googleapis.com/kubevirt-prow/reports/flakefinder/kubevirt/containerized-data-importer/flakefinder-2021-03-22-672h.html)

Reply:
ok so lets do it